dev-requirements.txt

```diff
@@ -1,4 +1,31 @@
+astroid==2.2.5
```

Comment:
Most of these dependencies should be in the requirements.txt. The `--r requirements.txt` file removed here allowed devs to run pip install -r dev-requirements.txt and have the dependencies in the requirements.txt installed.

Reply:
Noted. I will correct that now.